### PR TITLE
Modal fix ups

### PR DESCRIPTION
**Pull request by dillzhang**
Completed a couple of items in Asana
- When in mobile, 100% electronic item
- Hijack search to focus on our search (windows + mac)
- Sparkfun should be it's own link
- Images need to show half of the modal that's open
- Spacing between elements still change width changes
- Put Unknown for Lead Time if not written
- Add more tags

### CODE REVIEW

File: pages/electronics-library.tsx [hunk 1]
```diff
@@ -174,6 +175,43 @@ const Home = ({
     filterComponents();
   }, [searchCategoryTags, searchField]);
 
+  const metaTracker = useRef(false);
+
+  useEffect(() => {
```
Comment (poofytoo) [inline pages/electronics-library.tsx:180]: this piece of code is truly a work of art

File: styles/Page.module.scss [hunk 2]
```diff
@@ -159,6 +159,12 @@
   }
 }
 
+@media only screen and (min-width: 675px) {
```
Comment (poofytoo) [inline styles/Page.module.scss:162]: are you doing this in this way because of mobile first?

**Review by poofytoo — APPROVED** [review]
LGTM

File: pages/electronics-library.tsx (hunk 1 above)
Comment (dillzhang) [reply pages/electronics-library.tsx:180]: LOL I was definitely not sitting at my computer wondering why cmd+f wasn't working

File: styles/Page.module.scss (hunk 2 above)
Comment (dillzhang) [reply styles/Page.module.scss:162]: yeah

**Pull request merged**